docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -44,11 +38,16 @@ following steps:
    Static Pod manifests are written to `/etc/kubernetes/manifests`; the kubelet
    watches this directory for Pods to create on startup.
 
-   Once control plane Pods are up and running, the kubeadm init sequence can continue.
+   Once control plane Pods are up and running, the `kubeadm init` sequence can continue.
```

Comment:
Add something about dynamic kubelet configuration?

Reply:
sure -- feel free to! I was just improving the existing docs, we might wanna add some new, shiny v1.9 stuff as well